apps/server/src/modules/authentication/interface/jwt-payload.ts

```diff
@@ -1,7 +1,13 @@
-// TODO: should not get from this place over path
-import { ICurrentUser } from './user';
+export interface CreateJwtPayload {
+	accountId: string;
+	userId: string;
+	schoolId: string;
+	roles: string[];
```

Comment:
could you verify if this is really needed? since it doesn't seem to do its job.
what i expected:
all roles along with their permissions? (recursive)
what actually seem to happen: works non recursive

based on that thought, that this underlying logic is incorrect, the value in roles is not usable, apart from role name.

if that is the case one could drop roles entirely, and simply add roleName to show if a user is a teacher or whatever ..